logstash-core/lib/logstash/runner.rb

```diff
@@ -394,4 +353,7 @@ def trap_sigint
     end
   end
 
-end # class LogStash::Runner
+  def setting(key)
+    @settings.get_value(key)
+  end
+end
```

Comment:
another place https://github.com/elastic/logstash/pull/5251/files#diff-58bacb92415b22a57bdf05dc7459f937R272 with very similar code, I guess having more uniform access to settings would be nice to have.